order/buyer/receive/README.md

```diff
@@ -4,330 +4,334 @@ description: How to receive a purchase order response sent by the supplier
 
 # Receive an order response
 
-Tradecloud will send a purchase order response to the buyer when an order event has been triggered.
+Tradecloud sends purchase order responses to buyers when order events are triggered.
```

Comment:
I'd like to properly review this (it's quite tedious though).
However, it's not updated in the Gitbooks and most of it is out of scope for this PR. Can you extract it to a different PR so we can get the docs related to `originalPosition` out independently of that?

Reply:
reverted below ~ `Order header`

Reply:
Revied